scripts/SpaceCart.js

```diff
@@ -0,0 +1,25 @@
+import { transientState } from "TransientState.js"
+
+export const renderSpaceCart = async () => {
+    let response = await fetch ("https://localhost:8088/facilityMinerals?_expand=mineral&_expand=facility")
+    let facilityMinerals = await response.json()
+
+
+
+    let spaceCartHTML = ""
+
+    spaceCartHTML += `<div id = "space-cart">
+        <h1>Space Cart</h1>`
+        
+        html += facilityMinerals.map(item=> `${transientState.quantity} tons  of ${facilityMinerals.mineral.name} from ${facilityMinerals.facility.name}`)
+
```

Comment:
~~add a .join("") so it's adding a string instead of an array~~
Disregard this. The actual problems are listed below.

Reply:
We did away with quantity as well. For now, this is just 1 ton. We can update later if we need to.

Reply:
My first comment is maybe unnecessary. I think that instead of .map() ing, you're .find() ing.
One selection is made from radio buttons. You're finding the object in the facilityMinerals array whose id === transientState.facilityMineralId
Then you're using that object to fill out the string info.